charts/pega/values.yaml

```diff
@@ -20,6 +20,8 @@ global:
   # as an array of comma-separated strings using the certificatesSecrets parameter.
   certificatesSecrets: []
   certificates:
+  # If a storage class to be passed to the VolumeClaimTemplates in search and stream pods, it can be specified here:
+  storageClassName: ""
```

Comment:
I think it will be better if we add this line into values.yaml of pegaSearch as well